CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
The gruntfile is source changes too surely? It may have effects for people developing via bower link etc?
Where would you track those ops changes otherwise?